lib/src/boringssl/bindings/ffigen.yaml

```diff
@@ -0,0 +1,53 @@
+name: WebCryptoDartDL
+description: 'Bindings to src/webcrypto_dart_dl.h.'
+output: 'lib/src/boringssl/bindings/generated_bindings.dart'
+headers:
+  entry-points:
+    - src/webcrypto_dart_dl.h
+    - src/webcrypto.c
+compiler-opts: '-Ithird_party/dart-sdk/src/runtime/ -Ithird_party/boringssl/src/include -I/Library/Developer/CommandLineTools/usr/lib/clang/12.0.0/include'
+comments:
+  style: any
+  length: full
+sort: true
+typedef-map:
+  # TODO(dacoharkes): https://github.com/dart-lang/sdk/issues/42563 https://github.com/dart-lang/ffigen/issues/7
+  size_t: IntPtr
+macros:
+  include:
+    - nothing
+enums:
+  include:
+    - nothing
+unnamed-enums:
+  include:
+    - nothing
+globals:
+  include:
+    - nothing
+structs:
+  include:
+    - nothing
+  dependency-only: opaque
+functions:
+  include:
+    - webcrypto_dart_dl_attach_finalizer
+    - webcrypto_dart_dl_initialize
+    - webcrypto_lookup_symbol
```

Comment:
Maybe we shouldn't expose the `webcrypto_lookup_symbol` function. We could just do a manual lookup..

A library instance that contains `webcrypto_lookup_symbol` doesn't contain the other ones here.. Not that it matters, it's more of a nit...

Reply:
I slightly prefer to keep it, so that we don't have to write the signature manually.